src/coreclr/vm/gcenv.ee.cpp

```diff
@@ -388,8 +388,10 @@ bool GCToEEInterface::RefCountedHandleCallbacks(Object * pObject)
 #endif
 #ifdef FEATURE_COMWRAPPERS
     bool isRooted = false;
-    if (ComWrappersNative::HasManagedObjectComWrapper((OBJECTREF)pObject, &isRooted))
+    if (ComWrappersNative::IsRooted((OBJECTREF)pObject, &isRooted))
```

Comment:
This API is now a bit confusing. The former has a very clear contract for the return `bool` and the "out" `bool`. This API is now not clear.

Reply:
I adjusted the name to be more helpful.